RecoTracker/MkFitCore/BuildFile.xml

```diff
@@ -0,0 +1,8 @@
+<use name="rootsmatrix"/>
+<use name="json"/>
+<use name="tbb"/>
+<flags CXXFLAGS="-fopenmp-simd -Wno-error=strict-aliasing"/>
```

Comment:
Ignoring warnings is not really preferred, although we have some packages where it has been tolerated (not for this warning though).

IIRC the `-Wno-error=strict-aliasing` was added to get the `Ice` compiling with CMS' compilation flags. Since removing `Ice` is in the near term plan (but would imply physics changes that would be undesirable for this PR), perhaps it would be reasonable to tolerate ignoring the warning temporarily and remove it when `Ice` gets removed?

Reply:
Yes, that's what I'd say as well.

Reply:
I added a note to our todo document about Ice/ removal.